src/contexts/tvContext.jsx

```diff
@@ -11,7 +11,7 @@ const TvContextProvider = ({ children }) => {
   const [streams, setStreams] = useState([]);
   const [activeStream, setActiveStream] = useState(0);
   const [activeGame, setActiveGame] = useState(null);
-  const [activeLanguage, setActiveLanguage] = useState(DEFAULT_LANGUAGE);
+  const [activeLanguage, setActiveLanguage] = useState();
```

Comment:
Usually, useState hook takes a parameter to establish the state's initial value. Instead of leaving it empty (after you removed DEFAULT_LANGUAGE), I'd leave it like this:

```suggestion
  const [activeLanguage, setActiveLanguage] = useState(null);
```